src/test/CourseNavigation.test.jsx

```
@@ -0,0 +1,113 @@
+    expect(progressCircles).toHaveLength(4);
+  });
+
+  test("displays action buttons for courses", () => {
+    renderWithRouter(<CourseNavigation />);
+
+    // Should show "Continue" buttons for courses with progress
+    const continueButtons = screen.getAllByText("Continue");
+    expect(continueButtons.length).toBeGreaterThan(0);
+  });
+
+  test("expands course to show lessons when clicked", () => {
```

ESLint

[error] 'test' is not defined.

(no-undef)